src/types/index.ts

```diff
@@ -22,6 +22,28 @@ export type OfferListItem = {
   rating: number;
 }
 
+export type Offer = {
+  id: string;
+  title: string;
+  type: string;
+  price: number;
+  city: City;
+  location: Location;
+  isFavorite: boolean;
+  isPremium: boolean;
+  rating: number;
+  description: string;
+  bedrooms: number;
+  goods: [string];
+  host: {
+    name: string;
+    avatarUrl: string;
+    isPro: boolean;
+  };
+  images: [string];
```

Comment:
Твоя запись говорит что это будет массив с 1 значением и это строка, а у тебя же массив произвольной длины с элементами типа строка
```suggestion
  images: string[];
```